tests/test-cases/Group6-VIC-Machine/6-15-Syslog.robot

```diff
@@ -29,50 +30,62 @@ Get Remote PID
     &{proc-pids}=  Create Dictionary
     &{proc-hosts}=  Create Dictionary
 
+    ${vch-conn}=  Open Connection  ${vch-ip}
+    Login  root  password
     :FOR  ${proc}  IN  @{procs}
     \     ${pid}=  Get Remote PID  ${proc}
     \     Set To Dictionary  ${proc-pids}  ${proc}  ${pid}
     \     Set To Dictionary  ${proc-hosts}  ${proc}  ${vch-ip}
-
+    Close Connection
     Set To Dictionary  ${proc-hosts}  vic-init  Photon
 
+    ${rc}=  Run And Return Rc  docker %{VCH-PARAMS} ps -a
+    Should Be Equal As Integers  ${rc}  0
+
+    Run Regression Tests
+
+    ${pull}=  Run  docker %{VCH-PARAMS} pull ${busybox}
+    ${rc}  ${id}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d ${busybox} ls /
+    Should Be Equal As Integers  ${rc}  0
+    ${shortID}=  Get container shortID  ${id}
+
     ${syslog-conn}=  Open Connection  %{SYSLOG_SERVER}
     Login  %{SYSLOG_USER}  %{SYSLOG_PASSWD}
-
     ${out}=  Execute Command  cat ${SYSLOG_FILE}
+    Close Connection
+    Log  ${out}
+
     ${keys}=  Get Dictionary Keys  ${proc-pids}
     :FOR  ${proc}  IN  @{keys}
     \     ${pid}=  Get From Dictionary  ${proc-pids}  ${proc}
     \     ${host}=  Get From Dictionary  ${proc-hosts}  ${proc}
     \     Should Contain  ${out}  ${host} ${proc}[${pid}]:
 
-    ${rc}=  Run And Return Rc  docker %{VCH-PARAMS} ps -a
-    Should Be Equal As Integers  ${rc}  0
-
-    ${out}=  Execute Command  cat ${SYSLOG_FILE}
     ${pid}=  Get From Dictionary  ${proc-pids}  docker-engine-server
-    Should Contain  ${out}  ${vch-ip} docker-engine-server[${pid}]: Calling GET /v1.25/containers/json?all=1
+    Should Match Regexp  ${out}  ${vch-ip} docker-engine-server\\[${pid}\\]: Calling GET /v\\d.\\d{2}/containers/json\\?all\\=1
+
+    Should Match Regexp  ${out}  ${vch-ip} docker-engine-server\\[${pid}\\]: Calling POST /v\\d.\\d{2}/containers/create
+    Should Match Regexp  ${out}  ${vch-ip} docker-engine-server\\[${pid}\\]: Calling POST /v\\d.\\d{2}/images/create\\?fromImage\\=busybox\\&tag\\=latest
+    Should Match Regexp  ${out}  ${vch-ip} docker-engine-server\\[${pid}\\]: Calling POST /v\\d.\\d{2}/containers/\\w{64}/start
+    Should Match Regexp  ${out}  ${vch-ip} docker-engine-server\\[${pid}\\]: Calling POST /v\\d.\\d{2}/containers/\\w{64}/stop
+
+    Should Match Regexp  ${out}  ${vch-ip} docker-engine-server\\[${pid}\\]: Calling GET /v\\d.\\d{2}/images/json
+    Should Match Regexp  ${out}  ${vch-ip} docker-engine-server\\[${pid}\\]: Calling GET /v\\d.\\d{2}/containers/json
+    Should Match Regexp  ${out}  ${vch-ip} docker-engine-server\\[${pid}\\]: Calling GET /v\\d.\\d{2}/info
+
+    Should Match Regexp  ${out}  ${vch-ip} docker-engine-server\\[${pid}\\]: Calling DELETE /v\\d.\\d{2}/containers/\\w{64}
+    Should Match Regexp  ${out}  ${vch-ip} docker-engine-server\\[${pid}\\]: Calling DELETE /v\\d.\\d{2}/images/busybox
+
+    Should Match Regexp  ${out}  ${shortID} ${shortID}\\[1\\]: bin
```

Comment:
Can we match more than one line, considering we have had issues with incomplete attach output in the past?